common/lib/xmodule/xmodule/modulestore/split_mongo/split_draft.py

```diff
@@ -198,6 +198,8 @@ def delete_item(self, location, user_id, revision=None, skip_auto_publish=False,
                 branches_to_delete = [ModuleStoreEnum.BranchName.published, ModuleStoreEnum.BranchName.draft]
             elif revision is None:
                 branches_to_delete = [ModuleStoreEnum.BranchName.draft]
+                if location.category in DIRECT_ONLY_CATEGORIES:
```

Comment:
What is location here? I would expect it to be the video within the about component, but this code change implies (to me) that the location is actually the parent of the item being deleted.